app/views/estimate_flow/assets.html.slim

```diff
@@ -6,63 +6,7 @@
   h1.govuk-heading-xl class="govuk-!-margin-bottom-2" = t(".assets.legend")
   span.govuk-caption-m class="govuk-!-margin-bottom-9" = t(".assets.hint")
 
-  = form_for(@form,
-               url: wizard_path,
-               method: :put) do |form|
-
-    = form.govuk_error_summary t("generic.error_summary_title")
-
-    = form.govuk_fieldset legend: { text: t(".property.label") } do
-
-      .govuk-hint = t(".property.hint")
-
-      = render "shared/money_input",
-              form:,
-              field: :property_value,
-              width: 5,
-              label_text: t(".property.value.label")
-      = render "shared/money_input",
-              form:,
-              field: :property_mortgage,
-              width: 5,
-              label_text: t(".property.mortgage.label")
-      = form.govuk_number_field :property_percentage_owned,
-              width: 5,
-              suffix_text: "%",
-              label: { text: t(".property.percentage_owned.input") },
-              hint: { text: t(".property.percentage_owned.input_hint") }
-
-    = form.govuk_fieldset legend: { text: t(".savings.label") } do
-
-      .govuk-hint = t(".savings.hint")
-
-      = render "shared/money_input",
-            form:,
-            field: :savings,
-            width: 5,
-            label_text: t(".savings.input")
-
-    = form.govuk_fieldset legend: { text: t(".investments.label") } do
-
-      .govuk-hint = t(".investments.hint")
-
-      = render "shared/money_input",
-              form:,
-              field: :investments,
-              width: 5,
-              label_text: t(".investments.input")
-
-    = form.govuk_fieldset legend: { text: t(".valuables.label") } do
-
-      .govuk-hint = t(".valuables.hint")
-
-      = render "shared/money_input",
-               form:,
-               field: :valuables,
-               width: 5,
-               label_text: t(".valuables.input")
-
-    = form.govuk_submit t("generic.save_and_continue")
+  = render "estimate_flow/forms/assets", i18n_key: "assets", show_smod_content: true
```

Comment:
I'm a bit anti 'passing booleans into functions to change their behaviour' - is there a simpler way of breaking up the assets form so that we know that we're in partner-land (or not) to avoid this boolean?

Reply:
As discussed, no obvious better solution has emerged. This may be the lesser of multiple evils.